src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -796,25 +801,18 @@ void Deconvolution::createPrimitive() {
             auto outDummyDsc = getBaseMemDescAtOutputPort(0)->cloneWithNewDims(outDims);
             inDesc = MemoryDescUtils::convertToDnnlMemoryDesc(inDummyDsc);
             outDesc = MemoryDescUtils::convertToDnnlMemoryDesc(outDummyDsc);
-            if (withBiases) {
-                const VectorDims biasVecDims = getInputShapeAtPort(biasPort).getStaticDims();
-                auto biasDummyDsc = getBaseMemDescAtInputPort(biasPort)->cloneWithNewDims(biasVecDims);
-                biasDesc = MemoryDescUtils::convertToDnnlMemoryDesc(biasDummyDsc);
-            }
```

Comment:
So this old `if (withBiases) {` branch is worrying too much, fused bias always comes from constant node, with known static shape, right?

Reply:
Yes. Should be a const and dimEqualStrong in channel dimensin.